src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -251,13 +250,16 @@ const EntityTableToolbar = ({
       !(hideFilters.all && hideFilters.systemTypeFilter !== false) &&
       !hideFilters.systemTypeFilter,
   };
-
-  const exportConfig = useInventoryExport({
-    filters: {
-      ...activeFilters,
-      ...customFilters,
-    },
-  });
+  // I don't know how to fix this yet It calls chrome from the useExportApi
```

Comment:
What and how is it calling chrome?

Reply:
@bastilian the useAxiosWithPlatformInterceptors is called in this file src/components/InventoryTable/hooks/useInventoryExport/useExportApi.js

The path is
useInventoryExport in the EntityTableToolbar.js is calling useExportApi(); that calls useAxiosWithPlatformInterceptors();